src/main/java/org/jabref/migrations/PreferencesMigrations.java

```diff
@@ -328,6 +328,7 @@ protected static void upgradePreviewStyle(JabRefPreferences prefs) {
         String currentPreviewStyle = prefs.get(JabRefPreferences.PREVIEW_STYLE);
         String migratedStyle = currentPreviewStyle.replace("\\begin{review}<BR><BR><b>Review: </b> \\format[HTMLChars]{\\review} \\end{review}", "\\begin{comment}<BR><BR><b>Comment: </b> \\format[HTMLChars]{\\comment} \\end{comment}")
                                                   .replace("\\format[HTMLChars]{\\comment}", "\\format[Markdown,HTMLChars]{\\comment}")
+                                                  .replace("\\format[HTMLChars]{\\abstract}", "\\format[Markdown,HTMLChars]{\\abstract}")
```

Comment:
This is "only" the migration. The default preview style should also be updated. This would be a "more clean" solution as the migration only runs on start of JabRef. It shouldn't be necessary to update JabRef's internal defaults on start.

I wonder why Markdown ever worked as default?

Reply:
Default Preview is stored in the preferences